<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_setting</class>
 <widget class="QMainWindow" name="MainWindow_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开软件自动锁定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>自动拷贝链接</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>拷贝链接类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>监控剪贴板</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_CopyContent">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Markdown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Link</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_AutoLock">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>on.png</normaloff>on.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AutoCopy">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>on.png</normaloff>on.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MonitorClipboard">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>on.png</normaloff>on.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
